src/LogLevel.php

```diff
@@ -9,4 +9,10 @@ enum LogLevel
     case WARN; // Designates potentially harmful situations.
     case ERROR; // Designates error events that might still allow the application to continue running.
     case FATAL; // Designates very severe error events that will presumably lead the application to abort.
+
+    public static function getAllValues(): array
```

Comment:
LogLevel is a [Pure Enum](https://www.php.net/manual/en/language.enumerations.basics.php), So there is no `->value`